packages/dev/docs/pages/react-spectrum/theming.mdx

```diff
@@ -0,0 +1,107 @@
+<!-- Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import docs from 'docs:@react-types/provider/src/index.d.ts';
+import {Layout, TypeLink} from '@react-spectrum/docs';
+export default Layout;
+
+```tsx import
+import {Provider} from '@react-spectrum/provider';
+import {ActionButton} from '@react-spectrum/button';
+```
+
+---
+category: Concepts
+---
+
+# Theming
+
+This page describes how themes works in React Spectrum, including how applications adapt
+to operating system dark mode and touch scale.
+
+## Introduction
+
+React Spectrum is built to support theming. Colors, sizing, and spacing options can be customized
+through the use of CSS variables which are defined using the [Provider](Provider.html) component. Themes
+consist of definitions for these variables for light and dark [color schemes](https://spectrum.adobe.com/page/color/#Color-themes-and-modes),
+along with medium and large [platform scales](https://spectrum.adobe.com/page/platform-scale/).
+Themes can also be nested to allow different parts of an application to adopt a different appearance.
+
+## Color schemes
+
+React Spectrum supports multiple [color schemes](https://spectrum.adobe.com/page/color/#Color-themes-and-modes)
+within a theme, including light and dark mode. These correspond to color schemes found in popular operating systems. 
+By default, React Spectrum automatically switches between color schemes according to the operating system setting using the
+[prefers-color-scheme](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme)
+media query. This even updates automatically, so if a user changes their system's color scheme or it changes
+automatically (e.g. based on time of day), React Spectrum applications will update accordingly.
+
+We recommend that all React Spectrum applications support both light and dark mode. It can be very jarring to 
+use a light themed application when the rest of your applications are in dark mode, and visa versa. This is
+generally quite simple to support if you use Spectrum color variables rather than hard coded color values.
```

Comment:
Maybe `This can be accomplished by using Spectrum color variable rather than hard coded color values.`